helm/cluster-openstack/files/etc/patches/kube-apiserver.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "op": "add",
+        "path": "/spec/dnsPolicy",
+        "value": "ClusterFirstWithHostNet"
```

Comment:
just to learn, who is going to use this configuration? which component will be affected?

Reply:
the `kube-apiserver`, as the File has the name `kube-apiserver.json`

For a more details explanation, please read [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/#patches)